.github/workflows/main.yml

```diff
@@ -219,7 +219,7 @@ jobs:
     # Omit --all-targets on haiku because not all the tests build yet.
     # Temporarily disable this because the target appears to have breakage on nightly.
     #- run: cargo check -Z build-std --target x86_64-unknown-haiku --features=all-apis
-    - run: cargo check -Z build-std --target x86_64-uwp-windows-msvc --all-targets --features=all-apis
+    #- run: cargo check -Z build-std --target x86_64-uwp-windows-msvc --all-targets --features=all-apis
```

Comment:
Could you add a comment like the one on line 220??

Reply:
Fixed!